srcpkgs/mono-addins/patches/fix-delay-sign.patch

```diff
@@ -0,0 +1,21 @@
+Source: https://build.opensuse.org/package/view_file/openSUSE:Factory/mono-addins/fix-delay-sign.patch
+Upstream: No
+Reason: mono 5.0+ doesn't allow to install delay-signed dlls into GAC, so sign delay-signed dlls
+before adding it to GAC.
+--- Makefile.include	2016-07-27 01:26:26.000000000 +0300
++++ Makefile.include	2017-05-03 02:49:41.690295653 +0300
```

Comment:
The current version doesn't build because of [transition to -Np1](https://github.com/void-linux/void-packages/pull/31550).
```suggestion
--- a/Makefile.include	2016-07-27 01:26:26.000000000 +0300
+++ b/Makefile.include	2017-05-03 02:49:41.690295653 +0300
```